crates/astria-sequencer/local.env.example

```diff
@@ -0,0 +1,8 @@
+# Log level for the sequencer
+ASTRIA_SEQUENCER_LOG="astria_composer=info"
+
+# Address of the RPC server for the sequencer ABCI application
+ASTRIA_SEQUENCER_LISTEN_ADDR="http://127.0.0.1:26658"
+
+# Path to rocksdb
+ASTRIA_SEQUENCER_DATA_DIR="../"
```

Comment:
This is a misnomer: the code expects a file path, not a dir.


(also nit: newline)